### ISSUE TITLE
explanation of the incongruence

### ISSUE THREAD

**Issue by yozeng** [rivasiker/autocoalhmm#11]
Hi @rivasiker ,
If the proportion of inconsistent topology estimated by autocoalhmm is 30%, is this all due to ILS? Or is it possible that both ILS and introgression are contributing?

**Comment by rivasiker** [comment]
CoalHMM should ideally only be used when introgression is not present. In case there is introgression, you might be able to pick it from the posterior decoding because the two states incongruent with the species tree topology might be asymmetric, i.e., one of them might be more prevalent than the other. This is not expected if there is only ILS.

**Comment by yozeng** [comment]
Thank you for your reply! I asked this question because I've seen the work on distinguishing between ILS and introgression in your articles. So, is it right that the incongruent signals caused by introgression can also be detected by CoalHMM?

**Comment by rivasiker** [comment]
Yes, but the effect of introgression in the results produced by CoalHMM has never been systematically tested, so they should be interpreted with care. If there is introgression in the real data, the CoalHMM model would be misspecified and could lead to unforeseen biases in both the parameter estimation and the posterior decoding.

**Comment by yozeng** [comment]
Thanks! I've got it. By the way, I want to know whether the following topological correspondences are correct, especially V2 and V3:
V0/V1: (((sp1, sp2), sp3), sp4)
V2: (((sp1, sp3), sp2), sp4)
V3: (((sp2, sp3), sp1), sp4)

**Comment by rivasiker** [comment]
Correct!

**Comment by yozeng** [comment]
Hi @rivasiker ,
Sorry to disturb you again. I found that the final_table.HDF generated by this pipeline for some chromosomes has some rows with missing posterior probabilities. Is this normal and why is this happening?

![Image](https://github.com/user-attachments/assets/22db46b1-59ac-49d5-a4ef-caf2ad74bd5e)
